migrations/2015-10-13-fix-resources-metrics.js

```diff
@@ -0,0 +1,7 @@
+var result = db.dataset.update(
```

Comment:
It might be interesting to rename it to pass it prior to the one which split community resources?